tests/execute/duration/test.sh

```diff
@@ -3,23 +3,46 @@
 
 rlJournalStart
     rlPhaseStartSetup
-        rlRun "tmp=\$(mktemp -d)" 0 "Creating tmp directory"
         rlRun "pushd data"
     rlPhaseEnd
 
-    for method in tmt; do
-        rlPhaseStartTest "Test $method"
-            rlRun "tmt run -vfi $tmp -a execute -h $method test --name good" 0
-            rlRun "tmt run -vfi $tmp -a execute -h $method test --name long" 2
-            rlRun -s "tmt run --last report -fvvv" 2
-            rlAssertGrep "Maximum test time '3s' exceeded." $rlRun_LOG
-            rlAssertGrep "Adjust the test 'duration' attribute" $rlRun_LOG
-            rlAssertGrep "spec/tests.html#duration" $rlRun_LOG
-        rlPhaseEnd
+    for execute_method in tmt; do
+        for provision_method in $PROVISION_METHODS; do
```

Comment:
Perhaps add a `PROVISION_METHODS` default here as well so that it's possible to run the test simply as `./test.sh` without environment from metadata?

Reply:
Fixed in 45e9bd5c654d8c31645fc2e630f2d4ebcfdbd28d.